light_curves/light_curve_generator.md

```diff
@@ -6,9 +6,9 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.16.4
 kernelspec:
-  display_name: science_demo
+  display_name: notebook
   language: python
-  name: conda-env-science_demo-py
+  name: python3
```

Comment:
are these the new names on fornax?

Reply:
Sorry, I don't know, this all happens for me in the background when I open a notebook on Fornax.

Reply:
OK, we can change them, `python3` was the default for CI anyway, but it didn't make Fornax happy back then.